floss/api_hooks.py

```diff
@@ -122,7 +122,10 @@ def _fix_return(self, emu, return_address, return_addresses):
 
 class DemoHook:
     def __call__(
-        self, emu: viv_utils.emulator_drivers.EmulatorDriver, api: Tuple[str, Any, str, str, List], argv: List
```

Comment:
please only focus your changes on the minimum, this is likely due to a diff in linter configs?